travis/Dockerfile.i386

```diff
@@ -14,7 +14,7 @@ RUN yum install -y yum-utils && \
 
 # the shell wrapper takes care of enabling devtoolset and running a shell properly
 # unfortunately this is the easiest and most solid workaround to the limitations of the scl command
-COPY entrypoint.sh /
+COPY entrypoint-centos6.sh /entrypoint.sh
```

Comment:
There you have the `centos6` again in the name. This is good, though. The name of the base system should be consistent.

Reply:
This script would have to hardcode all ARCH if one want to name it `Dockerfile.i386.centos6`